src/testPane.ts

```diff
@@ -292,10 +305,19 @@ function getEnvironmentList(baseDirectory: string): string[] {
   // now we have a list of the UNITDATA.VCD files downstream of us
   // so turn this into a list of the enclosing directories only
   let returnList: string[] = [];
+  const workspaceFolderList = getWorkspaceFolderList();
   for (const file of fileList) {
     // note that glob always uses / as path separator ...
     if (!file.includes(".BAK")) {
-      returnList.push(path.dirname(file));
+      const candidatePath = path.dirname(file);
+      // we don't want to support users who add an enviro
+      // directory to the workspace - someone did this :()
+      if (!workspaceFolderList?.includes(candidatePath)) {
+        returnList.push(candidatePath);
+      } else {
+        vectorMessage (`Ignoring environment: ${candidatePath} ...`);
+        vectorMessage (`   environments should not be added to the workspace.`);
```

Comment:
"environments should not be opened as the workspace root"?

The current wording reads like "never open up a workspace that contains environments" (which is clearly wrong).